hugo/content/memory/_index.md

```diff
@@ -35,69 +33,74 @@ The first 2 bits in a 32-bit address.
 
 The first 8 bits in a 32-bit address.
 
-| *name*      | *prefix* |
-|-------------|----------|
-| ROM         | 0x00     |
-| RAM         | 0x40     |
-| MMIO        | 0xc0     |
-| MMIO TRNG   | 0xc0     |
-| MMIO TIMER  | 0xc1     |
-| MMIO UDS    | 0xc2     |
-| MMIO UART   | 0xc3     |
-| MMIO TOUCH  | 0xc4     |
-| MMIO FW_RAM | 0xd0     |
-| MMIO TK1    | 0xff     |
+| *name*      | *prefix*                        |
+|-------------|---------------------------------|
+| ROM         | 0x00                            |
+| RAM         | 0x40                            |
+| MMIO        | 0xc0                            |
+| MMIO TRNG   | 0xc0                            |
+| MMIO TIMER  | 0xc1                            |
+| MMIO UDS    | 0xc2                            |
+| MMIO UART   | 0xc3                            |
+| MMIO TOUCH  | 0xc4                            |
+| MMIO FW_RAM | 0xd0                            |
+| MMIO QEMU   | 0xfe Not used in real hardware! |
+| MMIO TK1    | 0xff                            |
```

Comment:
I'm sort of not fancy of the "MMIO" part. It is not I/O. It is internal functions. I know that it has become some sort of standard to call anything strictly not memory as MMIO. But do we gain anything by adding MMIO on front of everything here? This table explains what functionality is where at a high level. RAM is a function just as well as FW_RAM.

Side note: We could implement RAM and ROM using registers, would that mean that we should att MMIO there too?

Reply:
I removed MMIO completely from this table.